explainer.md

```diff
@@ -278,52 +159,66 @@ class CustomCheckbox extends HTMLElement {
   attributeChangedCallback(name, oldValue, newValue) {
     switch(name) {
     case "checked":
-      this.accessibleNode.checked = (newValue !== null);
+      this.shadowRoot.ariaChecked = (newValue !== null);
     }
   }
 
   connectedCallback() {
-    // When the custom checkbox is inserted in the page,
-    // ensure the checked state is in sync with the checked attribute.
-    this.accessibleNode.checked = this.hasAttribute("checked");
+    // Set the default semantics for the custom element
+    // when it is inserted in the page.
+    this.shadowRoot.role = "checkbox";
+    this.shadowRoot.checked = this.hasAttribute("checked");
   }
 }
 
 customElements.define("custom-checkbox", CustomCheckbox);
 ```
 
-From the perspective of the embedding page, a custom checkbox would now
-have this clean HTML, with the accessibility fully encapsulated into
-the implementation rather than exposed as a leaky abstraction:
+From the perspective of the embedding page, 
+a custom checkbox would now have this clean HTML, 
+with the accessibility fully encapsulated into the implementation 
+rather than exposed as a leaky abstraction:
 
 ```html
 <custom-checkbox checked>Receive promotional offers</custom-checkbox>
 ```
 
-Furthermore, writing Accessible Properties would allow specifying accessible relationships without requiring IDREFs,
-as authors can now pass object references.
-Currently attributes like aria-describedby and aria-activedescendant
-take one or more IDREFs, like this:
+The page author could override the default semantics using ARIA 
+exactly as they would with a built-in element:
 
 ```html
-<div role="listbox" aria-describedby="id1 id2" aria-activedescendant="id3">
+<!-- ARIA role overrides Shadow DOM role -->
+<custom-checkbox role="radio">
+```
+
+#### Use case 2: Setting relationship properties without needing to use IDREFs
+
+Straight reflection of ARIA properties 
+would reflect relationship attributes like `aria-labelledby` as strings:
+
+```js
+el.ariaDescribedBy = "id1";
 ```
 
-Accessible Properties would allow passing object references instead:
+We propose augmenting this API with non-reflected properties 
```

Comment:
maybe "We also propose augmenting this API with non-reflected properties"

Reply:
I feel like "also" is implied by "augmenting"?

Reply:
yeah i think how you have it is fine.